src/controllers/PendingTransactionController.ts

```diff
@@ -0,0 +1,52 @@
+
+import { Request, Response } from "express";
+import { sendJSONresponse } from "../common/Utils";
+import { Config } from "../common/Config";
+import { Error } from "mongoose";
```

Comment:
Are you use you need to include `Error` and `Promise`, in other classes, it's been resolved by itself

Reply:
Removed.